pkg/web/templates/openapi/openapi.json

```diff
@@ -11,12 +11,19 @@
       "name": "MIT Licensed",
       "url": "https://github.com/trisacrypto/trisa/blob/main/LICENSE"
     },
-    "version": "v0.23.0"
+    "version": "v{{ .Version }}"
   },
   "externalDocs": {
     "description": "TRISA Developer Documentation",
     "url": "https://trisa.dev"
   },
+  "servers": [
+    {
+      "url": "{{ .Origin }}",
+      "variables": {},
```

Comment:
I can't think of any `variables` that should be added here. Then again, I don't use the API docs a whole lot so there could be something others would find more useful that I'm not thinking of right now.